### PR TITLE
3670: Add get implementation timescales

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Solution.cs [hunk 1]
```diff
@@ -99,7 +99,16 @@ internal class Solution
         public Supplier Supplier { get; set; }
 
         /// <summary>
+<<<<<<< HEAD
```
Comment (RuthBerry96) [inline src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Solution.cs]: Whats the comment for head and the number below? This should be removed

File: src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Solution.cs [hunk 2]
```diff
@@ -116,8 +125,12 @@ internal class Solution
                 ? new List<string>()
                 : JsonConvert.DeserializeObject<IEnumerable<string>>(solutionResult.Features);
             IntegrationsUrl = solutionResult.IntegrationsUrl;
+<<<<<<< HEAD
```
Comment (RuthBerry96) [inline src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Solution.cs]: Same here

File: src/NHSD.BuyingCatalogue.Solutions.Application/Mapping/SolutionAutoMapperProfile.cs [hunk 3]
```diff
@@ -8,7 +8,11 @@
 using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetClientApplicationBySolutionId;
 using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetContactDetailBySolutionId;
 using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetHostingBySolutionId;
+<<<<<<< HEAD
```
Comment (RuthBerry96) [inline src/NHSD.BuyingCatalogue.Solutions.Application/Mapping/SolutionAutoMapperProfile.cs]: And here

File: tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/GetImplementationTimescalesBySolutionIdTests.cs [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+            _context.MockSolutionDetailRepository.Setup(r => r.GetImplementationTimescalesBySolutionIdAsync(_solutionId, _cancellationToken))
+            .ReturnsAsync(() => _mockResult.Object);
+            _context.MockSolutionRepository.Setup(r => r.CheckExists(_solutionId, _cancellationToken)).ReturnsAsync(() => _solutionExists);
+
+            _mockResult = new Mock<IImplementationTimescalesResult>();
+            _mockResult.Setup(m => m.Description).Returns(() => _implementationTimescalesDescription);
+        }
+
+        [TestCase("Some url")]
```
Comment (RuthBerry96) [inline tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/GetImplementationTimescalesBySolutionIdTests.cs]: Not testing a url. Change to description

File: tests/NHSD.BuyingCatalogue.Testing.Data/Entities/SolutionDetailEntity.cs [hunk 5]
```diff
@@ -23,10 +23,10 @@ public sealed class SolutionDetailEntity : EntityBase
 
         public string RoadMap { get; set; }
 
-        public string IntegrationsUrl { get; set; }
-
         public string RoadMapImageUrl { get; set; }
 
+        public string IntegrationsUrl { get; set; }
+
```
Comment (RuthBerry96) [inline tests/NHSD.BuyingCatalogue.Testing.Data/Entities/SolutionDetailEntity.cs]: Do you not want to add in your implementationtimescales?

**Comment by talawson05** [comment]
This PR should be showing green
The rerun button is missing for some reason so I triggered it manually:
https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_build/results?buildId=3133&view=results

**Pull request merged**